### ISSUE TITLE
Do not default `lang` attribute to null as this causes failing a11y audits

### ISSUE THREAD

**Issue by agubler** [dojo/widget-core#904]
**Bug**

Passing `null` as the default `lang` property in the `I18nMixin` causes a string of `null` to be applied as the `lang` attribute. This causes a11y audits to fail (using chrome dev tools). This should probably be defaulted to a zero length string.
